images/techdocs-cli/README.md

```diff
@@ -0,0 +1,15 @@
+# Backstage TechDocs Engineering Image
```

Comment:
I think we should consider what our long term plan is with docs, we generally have one devtools image for a specific type of input, golang, terraform, kubernetes.

I think it may make sense to have one devtools image for docs/lightweight-markup, and then in there have:
- static validation: markdown-lint, vale, [ltex](https://valentjn.github.io/ltex/)
- output generation for preview
- publishing: techdocs-cli (and mkdocs, rclone maybe)
- pipeline logic for validation and publishing (i.e. Makefile, Taskfile or similar)
And given this, I'm not sure if this specific image has much of a future, because I think we will soon need a devtools image that also includes the techdocs-cli.

Reply:
Anyway it is fine for now, I'm happy with this as is, just food for thought. We should think it through before we couple too many things to this though, but it will evolve invariably.

Reply:
We can add markdownlint and value to this. That is fine. I'll rename the image (again). The other tools will be added in a separate PR.

Reply:
You can keep it like this also for now, I don't think it is wrong entirely, and we need to explore the space a bit also.